cypress/support/commands.js

```diff
@@ -94,3 +94,17 @@ Cypress.Commands.add('testPagination', (selector, test) => {
   }
 })
 
+Cypress.Commands.add('sorting', (selector, order) => {
+  const collator = new Intl.Collator('en-US', {ignorePunctuation: true})
+  
+  cy.get(selector)
+  .then(($cell) => Cypress._.map($cell, (el) => el.innerText))
+  .then((list)=>{
+    var sortedList = [...list].sort((a, b) => order === "asc" ? collator.compare(a, b) : collator.compare(b, a));
```

Comment:
```suggestion
    const sortedList = [...list].sort((a, b) => order === "asc" ? collator.compare(a, b) : collator.compare(b, a));
```
Small suggestion, otherwise looks good to me.